code/modules/mob/living/simple_animal/hostile/retaliate/blind_wyvern.dm

```diff
@@ -0,0 +1,45 @@
+//cave dwelling dragon
+/mob/living/simple_animal/hostile/retaliate/blind_wyvern
+	name = "blind wyvern"
+	desc = "An eyeless reptile with huge wings. It looks straight out of myth."
+	icon = 'icons/mob/npc/cavern.dmi'
+	icon_state = "dragon" //icons by Kyres1
+	icon_living = "dragon"
+	icon_dead = "dragon_dead"
+	smart = TRUE
+	turns_per_move = 3
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+	a_intent = I_HURT
+	stop_automated_movement_when_pulled = 0
+	meat_type = /obj/item/reagent_containers/food/snacks/dwellermeat
+	mob_size = 12
```

Comment:
It's a big-ass wyvern. Should probably be closer to 20 (For reference, a bear is 17)